elements/table-title-field.js

```diff
@@ -115,15 +161,11 @@ function TitleField () {
     function renderButton () {
       if (state.editValue === state.title) {
         return html`
-          <button class="f6 white ttu bg-light-gray" onload=${attachListener}>
-            save
-          </button>
+          ${button('Save', { onload: attachListener })}
```

Comment:
this might have broken the "clicking outside" issue, i'll fix this before merging